### ISSUE TITLE
Style consistency

### ISSUE THREAD

**Issue by BrandonLMorris** [orlandoacevedo/MCGPU#49]
The project has a number of stylistic inconsistencies. While this is purely aesthetic, adhering to a style guide can improve readability, especially for developers new to the project.

I suggest we use Google's [C++ Style Guide](https://google.github.io/styleguide/cppguide.html), at least for the main issues:
- 2 space indentation (no tabs)
- 80 character maximum line length
- CamelCase variable/class names

Reformatting the project can be done automatically with the `clang-format` tool (note that this will show up as major changes in git). Simply run

`clang-format -style=Google -i **/*.cpp && clang-format -style=Google -i **/*.h`

in either the project root directory or in the `src` and `test` directory.

**Comment by orlandoacevedo** [comment]
That sounds good. If its that easy, then maybe just reformat the project prior to submitting your final push to Master.

**Comment by BrandonLMorris** [comment]
The reason I didn't was that all of my legitimate changes would be drowned out by the formatting changes in the diff. I thought about doing it in a separate branch, but that would create a host of merge conflicts.

**Comment by orlandoacevedo** [comment]
Makes sense. Push your legit changes first. We can then reformat after the fact.
